lib/ActivityStreams.js

```diff
@@ -18,6 +18,14 @@ XPCOMUtils.defineLazyGetter(this, "EventEmitter", function() {
   return EventEmitter;
 });
 
+XPCOMUtils.defineLazyServiceGetter(this, "windowWatcher",
```

Comment:
Actually, it might be better to do:

```javascript
Cu.import("resource://gre/modules/Services.jsm");
```

You can access the window watcher by calling `Services.ww` and `Services.wm`.
https://developer.mozilla.org/en-US/docs/Mozilla/JavaScript_code_modules/Services.jsm